flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/ContinuousHiveSplitEnumerator.java

```diff
@@ -93,6 +95,7 @@ public ContinuousHiveSplitEnumerator(
                         currentReadOffset,
                         seenPartitionsSinceOffset,
                         tablePath,
+                        flinkConf,
```

Comment:
As we only need `split.thread.num` in flinkConf, I think it maybe better to pass a single value representing the `split.thread.num instead of passing flinkconf.
It'll be more clear.

Reply:
Thanks for the valuable suggestion. But I have other considerations—— flinkConf can carry more info which maybe used by other features later. In this way, we need not add params one by one in functions. What do you think?

Reply:
ok, make sense to me.